### PR TITLE
Fixed issue #520

### LINKED ISSUE #520: Visual issues with EVO and simulate path
- [x] Simulate Path button looks like this after simulating a path 
image
- [ ] Fix the result box
image
- [ ] Slider goes down when adding requests 
image

### PR DESCRIPTION
**Pull request by Manarnz**
We fixed the EVO and EVO-related visual issues. This closes issue #520

### CODE REVIEW

File: leaf-ui/css/analysis_style.css [hunk 1]
```diff
@@ -342,36 +342,48 @@ select::-ms-expand {
...
-    border: none; 
-    outline: none;
-    float: left;
-  }
-  
-  .dropdown-container {
-    display:block
-  }
+.config-input {
+padding: 10px;
```
Comment (YesugenB) [inline leaf-ui/css/analysis_style.css]: Indent the lines inside all of the brackets

File: leaf-ui/css/style.css [hunk 2]
```diff
@@ -672,6 +672,7 @@ select > .select-placeholder {
 /* end of representive slider styling */
 
 .btn {
+  position: relative;
```
Comment (YesugenB) [inline leaf-ui/css/style.css:675]: Fix indent ?

File: leaf-ui/plugins/slider/nouislider.min.css [hunk 3]
```diff
@@ -1,4 +1,316 @@
 /*! nouislider - 8.2.1 - 2015-12-02 21:43:15 */
 
+.noUi-target,.noUi-target *
+{   -webkit-touch-callout:none;
+    -webkit-user-select:none;
+    -ms-touch-action:none;touch-action:none;
```
Comment (YesugenB) [inline leaf-ui/plugins/slider/nouislider.min.css:7]: put `touch-action:none` in a new line?

File: leaf-ui/plugins/slider/nouislider.min.css [hunk 4]
```diff
@@ -1,4 +1,316 @@
 /*! nouislider - 8.2.1 - 2015-12-02 21:43:15 */
 
+.noUi-target,.noUi-target *
+{   -webkit-touch-callout:none;
+    -webkit-user-select:none;
+    -ms-touch-action:none;touch-action:none;
+    -ms-user-select:none;
+    -moz-user-select:none;
+    -moz-box-sizing:border-box;
+    box-sizing:border-box
```
Comment (YesugenB) [inline leaf-ui/plugins/slider/nouislider.min.css]: Do we need a `- ` in the front here as well?

File: leaf-ui/plugins/slider/nouislider.min.css [hunk 5]
```diff
@@ -1,4 +1,316 @@
...
\ No newline at end of file
+}
+
+.noUi-target{
+    position:relative;
+    direction:ltr
+
```
Comment (YesugenB) [inline leaf-ui/plugins/slider/nouislider.min.css]: Delete extra lines in between brackets until end of file

File: leaf-ui/plugins/slider/nouislider.min.css [hunk 6]
```diff
@@ -1,4 +1,316 @@
...
+}
+
+.noUi-handle{
+    position:relative;
+    z-index:1;
+
+}
+
+.noUi-stacking .noUi-handle{
+    z-index:10
```
Comment (YesugenB) [inline leaf-ui/plugins/slider/nouislider.min.css]: Is the `z-index:1` for the previous noUI-handle being overwritten here?

File: leaf-ui/plugins/slider/nouislider.min.css [hunk 7]
```diff
@@ -1,4 +1,316 @@
...
+    -webkit-transition:left .3s,top .3s;transition:left .3s,top .3s
+
+}
+
+.noUi-state-drag *{
+    cursor:inherit!important
+
+}
+
+.noUi-base{
```
Comment (YesugenB) [inline leaf-ui/plugins/slider/nouislider.min.css]: There's a `noUi-base` in line 20 as well. Could these be combined?

File: leaf-ui/plugins/slider/nouislider.min.css [hunk 8]
```diff
@@ -1,4 +1,316 @@
...
+}
+
+.noUi-horizontal{
+    height:18px
+
+}
+
+.noUi-horizontal .noUi-handle{
+    width:34px;
+    height:28px;
```
Comment (YesugenB) [inline leaf-ui/plugins/slider/nouislider.min.css:59]: Is the previous height for `.noUi-horizontal ` in line 65 being overwritten? 

And is the top in line 73 being overwritten/overwriting the top in line 298

File: leaf-ui/plugins/slider/nouislider.min.css [hunk 9]
```diff
@@ -1,4 +1,316 @@
...
+    border:1px solid #D9D9D9;
+    border-radius:3px;
+    background:#fff;
+    padding:5px;
+    text-align:center
+
+}
+
+.noUi-horizontal .noUi-handle-lower .noUi-tooltip{
+    top:-32px
```
Comment (YesugenB) [inline leaf-ui/plugins/slider/nouislider.min.css]: Do you need` ; `at the end?

File: leaf-ui/plugins/slider/nouislider.min.css (hunk 4 above)
Comment (Manarnz) [reply leaf-ui/plugins/slider/nouislider.min.css]: I am not sure. I just made the file more readable. I didn't write it.

File: leaf-ui/plugins/slider/nouislider.min.css (hunk 8 above)
Comment (Manarnz) [reply leaf-ui/plugins/slider/nouislider.min.css:59]: I did not write the code, so I am not sure if I should touch it beyond making it readable for future use.

**Review by YesugenB — APPROVED** [review]

**Review by i-m-foster — APPROVED** [review]

**Review by amgrubb — APPROVED** [review]
Great job!

**Pull request merged**